File: CS/WebSite/DevExpress/ASPxSchedulerForms/AppointmentDragToolTip.ascx.cs
using System;
using System.Data;
using System.Configuration;
using System.Collections;
using System.Web;
using System.Web.Security;
using System.Web.UI;
using System.Web.UI.WebControls;
using System.Web.UI.WebControls.WebParts;
using System.Web.UI.HtmlControls;
using DevExpress.Web.ASPxScheduler;

public partial class UserForms_AppointmentDragToolTip : ASPxSchedulerToolTipBase {
    public override string ClassName { get { return "ASPxClientAppointmentDragTooltip"; } }
    public override bool ToolTipShowStem { get { return true; } }

    protected void Page_Load(object sender, EventArgs e) {
        DevExpress.Web.ASPxClasses.ASPxWebControl.RegisterBaseScript(Page);
    }
    protected override Control[] GetChildControls() {
        Control[] controls = new Control[] { lblInterval };
        return controls;
    }
}

File: VB/WebSite/DevExpress/ASPxSchedulerForms/AppointmentDragToolTip.ascx.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Data
Imports System.Configuration
Imports System.Collections
Imports System.Web
Imports System.Web.Security
Imports System.Web.UI
Imports System.Web.UI.WebControls
Imports System.Web.UI.WebControls.WebParts
Imports System.Web.UI.HtmlControls
Imports DevExpress.Web.ASPxScheduler

Partial Public Class UserForms_AppointmentDragToolTip
	Inherits ASPxSchedulerToolTipBase
	Public Overrides ReadOnly Property ClassName() As String
		Get
			Return "ASPxClientAppointmentDragTooltip"
		End Get
	End Property
	Public Overrides ReadOnly Property ToolTipShowStem() As Boolean
		Get
			Return True
		End Get
	End Property

	Protected Sub Page_Load(ByVal sender As Object, ByVal e As EventArgs)
		DevExpress.Web.ASPxClasses.ASPxWebControl.RegisterBaseScript(Page)
	End Sub
	Protected Overrides Function GetChildControls() As Control()
		Dim controls() As Control = { lblInterval }
		Return controls
	End Function
End Class
